test/spec/core/image.spec.js

```diff
@@ -70,6 +70,20 @@ describe("Image", function() {
 
   });
 
+  it("should extract the cdn_url from the vizjson", function(done) {
+
+    var vizjson = "http://documentation.cartodb.com/api/v2/viz/e7b04b62-b901-11e4-b0d7-0e018d66dc29/viz.json";
+
+    var image = cartodb.Image(vizjson);
+
+    image.getUrl(function(err, url) {
+      expect(image.options.cdn_url.http).toEqual("api.cartocdn.com");
```

Comment:
That's not the objective of this ticket, but I'll create a new one to allow using a custom CDN :)